Cargo.toml

```diff
@@ -70,7 +70,7 @@ futures-lite = "1"
 futures-rustls = "0.24"
 futures-util = "0.3.30"
 http-body-util = "0.1.1"
-hyper = { version = "1.2", default-features = false, features = ["server", "client", "http2"] }
+hyper = { version = "1.4.1", default-features = false, features = ["server", "client", "http2"] }
```

Comment:
Is there a reason to specify `1.4.1` rather than `1.4`? The old requirement was just on `1.2`.